langwatch/scripts/user-delete.sh

```
@@ -0,0 +1,527 @@
+SCHEMA=""
+if [[ "$DATABASE_URL" =~ schema=([^&]+) ]]; then
+    SCHEMA="${BASH_REMATCH[1]}"
+fi
+
+# Remove schema parameter from URL (psql doesn't understand it)
+PSQL_URL=$(echo "$DATABASE_URL" | sed 's/[&?]schema=[^&]*//')
+
+# Build psql options
+PSQL_OPTS=""
+if [ -n "$SCHEMA" ]; then
+    PSQL_OPTS="-c \"SET search_path TO $SCHEMA, public;\""
```

Shellcheck (0.11.0)

[warning] PSQL_OPTS appears unused. Verify use (or export if used externally).

(SC2034)